manim/mobject/types/vectorized_mobject.py

```diff
@@ -1726,6 +1726,44 @@ def construct(self):
                 vdict_using_zip.shift(1.5 * RIGHT)
                 self.play(Create(vdict_using_zip))
                 self.wait()
+
+    .. manim:: VDictExample
+        :save_last_frame:
+
+        class VDictExample(Scene):
+            def construct(self):
+                n = 35  # Number of dots in the Grid is 35^2
+                grid = VDict()
+                boolDict = {}  # Standard Dictionary keeping track of which dots are white.
+                for r in np.arange(-(n - 1) / 2, (n + 1) / 2, 1):
+                    for c in np.arange(-(n - 1) / 2, (n + 1) / 2, 1):
+                        key = f"{int(r)},{int(c)}"
```

Comment:
Is there a particular reason why the keys are converted to strings? I'd prefer if simply the tuple `(int(r), int(c))` was used. This would remove the need for pretty much all of the f-strings in the later part of the code.

Reply:
Honestly, I never thought about using a tuple. I just got use to using f-strings as keys for dictionaries. The latest changes to my pull request should have phased out all the f-strings for tuples. Thank you for your suggestion.